private/bufpkg/bufprotosource/files.go

```diff
@@ -50,8 +51,11 @@ func newFiles[F InputFile](
 		}
 		return files, nil
 	}
-
-	chunks := slicesext.ToChunks(indexedInputFiles, chunkSize)
+	// slices.Chunk panics if given a chunk size less than 1 - allocate a single chunk in this case.
```

Comment:
This is one difference between the previous `slicesext.ToChunks` implementation and the stdlib one, although it doesn't seem too bad in practice. If desired, I could re-add the method to slicesext and add this handling there but still re-use the stdlib method.

Reply:
Let's re-add the method to `slicesext` with the handling encapsulated since the `slicesext` package is staying, so I think it makes sense to minimize the impact of this.

Reply:
Added back in b52fd39bf and re-added tests.